fictadvisor-api/src/v2/utils/documentation/user/updateStudent.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiDocumentationParams } from '../decorators';
+import { FullStudentResponse } from '@fictadvisor/utils';
+import { DefaultForbiddenResponse, DefaultUnauthorizedResponse } from '../defaultResponses';
+
+export const UserDocumentationUpdateStudent: ApiDocumentationParams = {
+  isAuth: true,
+  ok: {
+    type: FullStudentResponse,
```

Comment:
Response type has an id field, while the endpoint doesn't return it (check StudentMapper.updateStudent)